src/ring/middleware/oauth2.clj

```diff
@@ -44,8 +44,9 @@
     n))
 
 (defn- format-access-token
-  [{{:keys [access_token expires_in refresh_token id_token]} :body :as r}]
-  (-> {:token access_token}
+  [{{:keys [access_token expires_in refresh_token id_token] :as body} :body :as _r}]
```

Comment:
The `:as _r` doesn't appear to be doing anything, so can be removed.

Reply:
done :)